src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
```

Comment:
Common mistakes 
You must have a few separated services and call step by step in main(). And results of previous method it is a argument for next.
1 Read from file and save to list results
2 convert the result from a file into a list of operations
3 process the operations list
4 create report
5 write report to file

Reply:
not fixed